packages/auth/src/mfa/assertions/totp.test.ts

```diff
@@ -127,6 +131,7 @@ describe('core/mfa/assertions/totp/TotpMultiFactorGenerator', () => {
 
 describe('core/mfa/totp/assertions/TotpMultiFactorAssertionImpl', () => {
   let auth: TestAuth;
+  let user: UserInternal;
```

Comment:
Is this needed in addition to L106?

Reply:
i think it is needed because L106 is in a different describe block.